rpm/timed-qt5.spec

```diff
@@ -73,13 +73,13 @@ rm -rf %{buildroot}
 
 # The file %{buildroot}%{_userunitdir}/%{name}.service is installed by make install
 install -d %{buildroot}%{_userunitdir}/pre-user-session.target.wants/
-ln -s ../%{name}.service %{buildroot}%{_userunitdir}/pre-user-session.target.wants/%{name}.service
+ln -s ../timed.service %{buildroot}%{_userunitdir}/pre-user-session.target.wants/%{name}.service
```

Comment:
```suggestion
ln -s ../timed.service %{buildroot}%{_userunitdir}/pre-user-session.target.wants/timed.service
```

Reply:
Please give also 

add-oneshot --now setcaps-%{name}.sh &
systemctl-user restart %{name}.service || :

From the %post section.